README.md

```diff
@@ -92,6 +101,23 @@ Terminologies used in this project.
 | [/test](./test)                                   | Test Framework and Test Suite including e2e test and functional test           |
 | [/utils](./utils)                                 | Utilities used for build, setup, CI/CD                                         |
 
+
+
+## Build <a name = "build"> </a>
+Run following command on root directory to build the rust contracts. The built wasm files will be available in the artifacts directory in the root.
+```
+./scripts/optimize-cosmwasm.sh
+```
+To build the java contracts run following command.
+```
+make optimize-jar
+```
+To build all contracts using docker we can run following commands.
+```
+make build-builder-img
+make optimize-build
```

Comment:
As the procedure involves the building of the xcall contract from xcall submodule, should we provide instructions to init xCall submodule?

Reply:
we have mentioned that in installation
 `git submodule init
git submodule update --remote`